scripts/lv_conf_internal_gen.py

```diff
@@ -197,6 +197,12 @@
     #define LV_USE_THORVG  (LV_USE_THORVG_INTERNAL || LV_USE_THORVG_EXTERNAL)
 #endif
 
+#if LV_USE_OS
+    #if (LV_USE_FREETYPE || LV_USE_THORVG) && LV_DRAW_THREAD_STACK_SIZE < (24 * 1024)
+        #warning "LV_DRAW_THREAD_STACK_SIZE is too small for FreeType or ThorVG. It recommended to set it to 32KB or more."
```

Comment:
typo: It **'s** recommended to ... But maybe better for this:

```suggestion
        #warning "Increase LV_DRAW_THREAD_STACK_SIZE to at least 32KB for FreeType or ThorVG."
```

Reply:
Done